fighters/demon/src/acmd/smashes.rs

```diff
@@ -31,22 +32,23 @@ unsafe fn demon_attack_s4_game(fighter: &mut L2CAgentBase) {
     FT_MOTION_RATE(fighter, 1.0);
     frame(lua_state, 25.0);
     if is_excute(fighter) {
+        FT_DESIRED_RATE(fighter, 28.0-25.0, 4.0);
```

Comment:
line 35 `FT_DESIRED_RATE` should be cut and pasted in place of like 32 `FT_MOTION_RATE`